easybuild/easyconfigs/f/FIAT/FIAT-1.1-ictce-5.2.0-Python-2.7.3.eb

```diff
@@ -10,7 +10,8 @@ arbitrary order instances of Jacobi-type quadrature rules on the same element sh
 
 toolchain = {'name': 'ictce', 'version': '5.2.0'}
 
-source_urls = ['https://launchpad.net/%(namelower)s/%(version_major_minor)s.x/release-%(version_major_minor)s/+download/']
+source_urls = [
```

Comment:
@boegel what do you think about this? We can make an exception for this by appending `#noqa`?